src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
+@import "variables";
+@import "card";
+@import "stars";
+
 body {
   margin: 0;
 }
```

Comment:
It's better to move this style to a separate file to keep your main file clean and clear